app/src/main/java/com/zeapo/pwdstore/PasswordFragment.kt

```diff
@@ -38,6 +41,7 @@ class PasswordFragment : Fragment() {
     private lateinit var recyclerAdapter: PasswordItemRecyclerAdapter
     private lateinit var listener: OnFragmentInteractionListener
 
+    private var settings: SharedPreferences? = null
```

Comment:
Maybe make this `lateinit` ?

Reply:
Hm yeah should be fine. Will make all the changes.